src/GUI_draft/templates/landing_page.html

```diff
@@ -3,40 +3,39 @@
 
 <!-- WRITE HTML CODE WITHIN THESE block content TAGS -->
 {% block content %}
-
-<!--
-<div class="grid-x">
-<div class="cell">
-    {% import "progression_bar.html" as progression_bar %}
-            this is just an example, you can complete the dict with all the steps
-    {{ progression_bar.input(step_idx=1, step_map=step_map) }}
-
-</div>
-</div>
--->
-
+<body>
```

Comment:
body tag is already present in `base.html`, it should not be used twice

Reply:
I added it because I have a script to run. Scripts must always be put before the end of a body tag right. Where should I put the script code snippet then?

Reply:
Edit: I got it. Ignore my question above.